frontend/src/app/main/main/main.component.ts

```diff
@@ -14,30 +20,126 @@ import { UntilDestroy, untilDestroyed } from '@ngneat/until-destroy';
 })
 export class MainComponent implements OnInit {
   public accounts$!: Observable<IAccount[]>;
-  public transactions$!: Observable<ITransaction[]>;
+  public transactions!: ITransaction[];
+  public showDeletedTransactionNotification: boolean = false;
+  private selectedAccountId!: string;
+  private selectedAccountCurrency!: ICurrency;
 
   constructor(
     private accountsService: AccountService,
-    public transactionsService: TransactionService
+    public transactionsService: TransactionService,
+    private matDialog: MatDialog
   ) {}
 
   public ngOnInit(): void {
-    this.accounts$ = this.accountsService.getAccounts();
+    this.getAccounts();
```

Comment:
you send this request twice. I think this one can be removed

Reply:
Not really.  During initialization I only send this request once.
![image](https://user-images.githubusercontent.com/55427679/163580934-60f85ade-7f90-4e2e-b4ab-fa15b0da8f0e.png)
Observable  which I get when I call getAccountsChangeObservable only emits value only when account get updated or deleted.
![image](https://user-images.githubusercontent.com/55427679/163581439-38f742a9-87be-4d5d-a939-00b100dac01a.png)
So I send request initially and after that I send request only after some account was updated or deleted.